inst/postgres/10-values.sql

```diff
@@ -1,2 +0,0 @@
-INSERT INTO method (name)
-VALUES ('simple'), ('blocked');
```

Comment:
Możliwe, że można zastąpić: 
`INSERT INTO study (method)
VALUES ('simple'), ('minimise_pocock');` 

I testować potem na tych metodach

Reply:
This file was entirely deleted - there is no "method" table no more.